open_policy_agent/manifest.json

```
@@ -0,0 +1,39 @@
+{
+  "display_name": "open-policy-agent",
+  "maintainer": "ara.pulido@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "open_policy_agent",
+  "metric_prefix": "open_policy_agent.",
+  "metric_to_check": "open_policy_agent.policies",
+  "creates_events": false,
+  "short_description": "OPA integration",
+  "guid": "a68c72bd-a16a-4fcf-8911-43575dd722b9",
+  "support": "contrib",
+  "supported_os": [
+    "linux"
+  ],
+  "public_title": "Open-Policy-Agent",
```

```suggestion
  "public_title": "Open Policy Agent",
```

This is used for the tile name. We should keep consistent with branding